File: javascript/soma_vetores.js
const n = parseInt(prompt("Quantos valores vai ter cada vetor?"));

let A = [], B = [], C = [];

console.log("Digite os valores do vetor A:");
for(i = 0; i < n; i++) {
    A[i] = parseInt(prompt());
}

console.log("Digite os valores do vetor B:");
for(i = 0; i < n; i++) {
    B[i] = parseInt(prompt());
}

console.log("VETOR RESULTANTE:");
for(i = 0; i < n; i++) {
    C[i] = A[i] + B[i];
    console.log(C[i]);
}

File: python/soma_vetores.py
n = int(input("Quantos valores vai ter cada vetor? "))

a = []
b = []
c = []

print("Digite os valores do vetor A: ")
for i in range(0, n):
    a.append(int(input()))

print("Digite os valores do vetor B: ")
for i in range(0, n):
    b.append(int(input()))

print("VETOR RESULTANTE:")
for i in range(0, n):
    c.append(a[i] + b[i])
    print(c[i])
